store/CSDK/CMakeLists.txt

```diff
@@ -1,17 +1,17 @@
-cmake_minimum_required (VERSION 2.8)
-project (CJDK)
+cmake_minimum_required(VERSION 2.8)
```

Comment:
Shouldn't add license header?

Reply:
I think no need to add license header in this file.
CMakeLists.txt like pom.xml, it's not code file. 
Tensorflow and caffe project also didn't add license header in CMakeLists.txt .

Reply:
ok, added